modules/python/src/args.py

```diff
@@ -35,9 +35,15 @@ def iam_groups(self):
         s3 = boto3.resource('s3')
         bucket = s3.Bucket('${bucket_name}')
         groups = []
-        for group_json in bucket.objects.all():
-            users = helper.json_loads(group_json.get()['Body'].read().decode('utf-8'))
-            group_name = splitext(basename(group_json.key))[0]
+        for s3_group in bucket.objects.all():
```

Comment:
It would be better if we just list out all the *.json files and select the groups from them, this way we don't need to check the extensions later on.

Reply:
Look like the current S3 API support Prefix only, https://docs.aws.amazon.com/AmazonS3/latest/API/v2-RESTBucketGET.html

We can do filter using python collection api as well, but the benefit not better than current implementation, since it also get all items in S3 to do the filtering.